### PR TITLE
set protocol value following the spec

### PR DESCRIPTION
**Pull request by northteam**
#1255 

Set protocol/relayProtocol of RtcIceCandidateStats following the [specification](https://www.w3.org/TR/webrtc-stats/#icecandidate-dict*).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/source/Ice/IceUtils.h [hunk 1]
```diff
@@ -21,6 +21,10 @@ extern "C" {
 #define ICE_URL_TRANSPORT_UDP      "transport=udp"
 #define ICE_URL_TRANSPORT_TCP      "transport=tcp"
 
+#define ICE_TRANSPORT_TYPE_UDP "UDP"
```
Comment (MushMal) [inline src/source/Ice/IceUtils.h:24]: Wondering if these should be lowercase?

Comment (disa6302) [reply src/source/Ice/IceUtils.h:24]: Yes. Let us make it lowercase.

**Comment by disa6302** [comment]
@northteam , 

Thank you for your PR.Can you please open this PR by rebasing off `origin/develop`? We do not accept PRs opened on Master anymore. 

This is what needs to be done:
1. Checkout `develop`. 
2. Rebase your changes off top of develop
3. Open a new PR to with `awslabs:develop` as base branch

**Comment by northteam** [comment]
Thank you for your reply. I'll open a new PR to develop.

**Pull request closed without merging**